.travis.yml

```diff
@@ -28,3 +28,4 @@ addons:
     packages:
     - git-annex
     - rpm
+    - python-rpm
```

Comment:
Is this still needed given the revert back to using subprocess?

Reply:
oop, yeah needs to be removed.